RNTester/js/SwipeableListViewExample.js

```diff
@@ -100,45 +94,45 @@ var SwipeableListViewSimpleExample = createReactClass({
         </View>
       </TouchableHighlight>
     );
-  },
+  };
 
-  _genDataSource: function(pressData: {[key: number]: boolean}): Array<any> {
-    var dataBlob = {};
-    var sectionIDs = ['Section 0'];
-    var rowIDs = [[]];
+  _genDataSource(pressData: {[key: number]: boolean}): Array<any> {
```

Comment:
The return type here might be overly generic, given that we know the types of each of the elements in the array. 

```typescript
const dataBlob: {[key: string]: { id: string, text: string }} = {}
const sectionIDs: Array<string> = ['Section 0'];
const rowIds: Array<Array<string>> = [[]]
```

I wonder if it's better to remove it. Flow is smart enough to infer the return type.

Reply:
Flow seemed fine with removing it